### PR TITLE
Unified the code for bsc and ethereum-rpc containers

### PR DESCRIPTION
**Pull request by AjmalAhamedKodakkat**

### CODE REVIEW

File: ethereum-rpc/src/constructors/ethereum/compiler.js [hunk 1]
```diff
@@ -2,7 +2,9 @@ const path = require("path");
 const fs = require("fs");
 const solc = require("solc");
 const ethers = require("ethers");
-const web3 = require("@/constructors/web3");
+//const web3 = require("@/constructors/web3");
+const web3Connection = require('@/constructors/web3')
+const {web3,bscWeb3} = web3Connection;
```
Comment (srslafazan) [inline ethereum-rpc/src/constructors/ethereum/compiler.js:7]: we may be able to keep these together by having both use the HttpProvider

File: ethereum-rpc/src/constructors/ethereum/index.js [hunk 2]
```diff
@@ -3,7 +3,8 @@ const path = require('path')
 const checksum = require('ethereum-checksum-address')
 // const decoder = require('abi-decoder')
 
-const web3 = require('@/constructors/web3')
+const web3Connection = require('@/constructors/web3')
+const {web3,bscWeb3} = web3Connection;
```
Comment (srslafazan) [inline ethereum-rpc/src/constructors/ethereum/index.js:7]: Let's export just web3, and in the case of binance smart chain, export a different web3 instance

File: ethereum-rpc/src/constructors/web3.js [hunk 3]
```diff
@@ -24,4 +24,9 @@ const options = {
   },
 };
 
-module.exports = new Web3(new Web3WsProvider(process.env.ETHEREUM_WSS_ADDR || 'http://127.0.0.1:8546', options))
+const web3 = new Web3(new Web3WsProvider(process.env.ETHEREUM_WSS_ADDR || 'http://127.0.0.1:8546', options))
+const bscWeb3 = new Web3(process.env.BINANCE_SMART_CHAIN_HTTPS_ADDR || 'http://127.0.0.1:8546', options)
+module.exports = {
+web3,
+bscWeb3
+}
```
Comment (srslafazan) [inline ethereum-rpc/src/constructors/web3.js:32]: Here, is it possible to use HttpProvider for both?

Comment (AjmalAhamedKodakkat) [reply ethereum-rpc/src/constructors/web3.js:32]: Sure , I will try to switch both to http

File: ethereum-rpc/src/constructors/ethereum/index.js (hunk 2 above)
Comment (AjmalAhamedKodakkat) [reply ethereum-rpc/src/constructors/ethereum/index.js:7]: ok , let I will take another web3 instance for bsc

**Review by srslafazan — APPROVED** [review]
LGTM! -- let's remove the remaining binance-smart-chain-rpc files and merge.

**Pull request merged**